samigo/samigo-app/src/webapp/js/saveForm.js

```diff
@@ -179,7 +174,7 @@ function SaveFormContentAsync(toUrl, formId, buttonName, updateVar, updateVar2,
         // when the request is done the scope of the function can be garbage collected...
     }
 
-    if (doautosave && counter > 1) {
+    if (doautosave) {
```

Comment:
doautosave is always true, it seems. It gets set to true and nothing else touches it.

Reply:
Can I add a commit onto this, just doing some cleanup? Variables that are only accessed once, formatting all over the place. I can shorten this file and make it a lot clearer.

Reply:
I thought it was overridden from a JSP .... but I can't find any overrides. So sure, let's remove it. But I'd rather not dramatically re-format this JS .... it's super sensitive and this code is super prone to regressions

Reply:
Yeah, looking at this it doesn't look like it's used, it looks like it has always been true. It's not passed in from deliverAssessment (SaveFormContentAsync) and if this is called it's going to be true. This is really hard to read code for sure.

Reply:
I somewhat expected Codacy to catch this. It doesn't but it also really doesn't like this file. https://app.codacy.com/manual/sakaiproject/sakai/file/42082279127/issues/source?bid=16464702&fileBranchId=16464702